common/kvstore/Batch.go

```diff
@@ -0,0 +1,14 @@
+package kvstore
+
+// Batch is a collection of key / value pairs that can be written atomically to a database.
```

Comment:
nit: `can be written` -> `will be written` atomically?

Mention that the Batch is not thread safe